2-mandatory/3-shopping-list.js

```diff
@@ -19,8 +19,25 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
-}
```

Comment:
This looks good to me 👍 
As an extra challenge, can you re-write this using the `filter` array method?